package.json

```diff
@@ -54,9 +54,11 @@
     "@patternfly/patternfly": "4.135.2",
     "@patternfly/react-core": "4.157.3",
     "@patternfly/react-styles": "4.11.16",
+    "@patternfly/react-icons": "4.11.17",
     "date-fns": "2.24.0",
     "react": "16.14.0",
     "react-dom": "16.14.0",
+    "react-dropzone": "11.4.2",
```

Comment:
This I don't understand, and agree with -- nothing in our own code uses react-dropzone, so we shouldn't directly depend on it. Looking at my
[package-lock.json](https://github.com/cockpit-project/cockpit-ostree/files/7226063/package-lock.json.txt), I see that it is a dependency of `@patternfly/react-core`, and it wants version 9.0.0, not 11.4. So this seems actively wrong to me?

Can you compare your package-lock.json (without this PR) to mine, and check if there is some meaningful difference?

I'm running in cockpit/tasks container, with npm-6.14.15-1.14.17.6.1.fc34.x86_64 and nodejs-14.17.6-1.fc34.x86_64

Reply:
If I remove it, it fails like this:
```
ERROR in ./node_modules/@patternfly/react-core/dist/esm/components/FileUpload/FileUpload.js 3:0-38
Module not found: Error: Can't resolve 'react-dropzone' in '/home/mmarusak/cockpit/cockpit-ostree/node_modules/@patternfly/react-core/dist/esm/components/FileUpload'
```
npm version `7.21.1`
node version `16.9.1`

Reply:
That's Fedora 35 ? Does it actually install react-dropzone, and how does it look like in your package-lock.json?

Reply:
Does that affect c-{podman,machines} and cockpit itself also? We don't use the `FileUpload` component in any of our projects, so ATM I don't see what's special about ostree

Reply:
It does not affect any other component. I am not sure, why react-icons need to be specified explicitly either. I was trying to understand what's the difference between ostree and podman, but could not find anything noteworthy.

Reply:
We do use react-icons explicitly:
```
src/app.jsx:import { ExclamationCircleIcon, OutlinedCheckCircleIcon } from '@patternfly/react-icons';
src/changeRemoteModal.jsx:import { PencilAltIcon, AddCircleOIcon } from '@patternfly/react-icons';
```
Thus we should have an explicitly versioned `dependencies` entry for it. That's not the root cause for the build failure (@jelly had some good ideas about our webpack config's `module.export.resolve` entry?), but it's still the right thing to do.

Reply:
ha, now I changed branch in podman (to something new I presume) and it fails as well:
```
ERROR in ./node_modules/@patternfly/react-core/dist/esm/components/FileUpload/FileUpload.js 3:0-38
Module not found: Error: Can't resolve 'react-dropzone' in '/home/mmarusak/cockpit/cockpit-podman/node_modules/@patternfly/react-core/dist/esm/components/FileUpload'

```